### ISSUE TITLE
Pokebuddy

### ISSUE THREAD

**Issue by futi666** [finndev/PokeBuddy#336]
hello guys, anyone has any idea when the pokebuddy is online again? 
cheers

**Comment by TheValiden** [comment]
in 6 hours or tomorrow

**Comment by futi666** [comment]
thank you for ur quick answer

**Comment by Sudiddo** [comment]
can't wait for it.. oh gosh, so excited :)))

**Comment by TheValiden** [comment]
just an idea np :)

**Comment by TheValiden** [comment]
https://www.reddit.com/live/xdkgkncepvcq

**Comment by craigyjr** [comment]
good work to all the devs getting the problem solved i know pokebuddy not been updated but i can wait, as a wheelchair user its a great help to me to have a bot like this i just traid necrobot but i cudnt get it to farm or catch again just wanna say good work

**Comment by Salvo00** [comment]
Ma quindi il bot domani tornerà a funzionare non ciò capito un gran che..

**Comment by bertje123** [comment]
there were some updates 2 min ago. is it working again?

**Comment by XMGamingMaster** [comment]
Yes, it works.

**Comment by nafmal** [comment]
Thanks for the great working!

**Comment by saintv** [comment]
I got ban right away using the updated bot.

**Comment by JackSbirrow** [comment]
Does PokeBuddy work now? Do i need to download it again?

**Comment by 404unknown** [comment]
@saintv what do you mean of "banned"? Do you mean soft banned or what?

**Comment by saintv** [comment]
Yeah i got a soft ban, and my eevees got transferred when i choose not to transfer them.

**Comment by 404unknown** [comment]
Yea I guess the transfer system is still quite glitchy
